playbook/app/pb_kits/playbook/pb_selectable_card/_selectable_card.jsx

```diff
@@ -55,7 +57,9 @@ const SelectableCard = ({
     { 'checked': checked,
       'disabled': disabled,
       'enabled': !disabled }),
-  globalProps(props), className)
```

Comment:
Is dropping `globalProps` cool here?

Reply:
I see it's invoked below on line 98 but I think it should be here as well, otherwise global props won't make it into the `classes` variable which gets passed to the outer div on line 84.

Reply:
Putting global props in two places is what was causing the padding and margin props to apply twice, both to the outer div and to the label that actually displays.  This means that you get twice as much space as you intended.

Global props looks like it includes dark props and spacing props. I accounted for dark manually since I didn't want to put spacing on the outer element.

 Open to better ways to handle, let me know what you guys think

Reply:
Would it create problems to leave `globalProps` off the inner `label` element and leave it only on the outermost div?

Reply:
@bh247484 The reason for putting global props on the inner `label` is that is the element you actually see when the kit is rendered. If you put global props only on the outermost `div`, then any padding applies outside the border and not inside, as you'd likely expect when using the kit